src/main/java/run/halo/app/cache/LevelCacheStore.java

```diff
@@ -32,8 +32,9 @@
 
     private Timer timer;
 
-    @Autowired
-    private HaloProperties haloProperties;
+    public LevelCacheStore(HaloProperties haloProperties) {
```

Comment:
The unused import `import org.springframework.beans.factory.annotation.Autowired;` should be removed as well.

Reply:
fixed